db/migrate/20210512180943_change_column_name.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeColumnName < ActiveRecord::Migration[5.2]
+  def change
```

Comment:
Clutch. Awesome you were able to implement a new migration really quickly after yesterday.